doc/nrf/libraries/caf/sensor_sampler.rst

```diff
@@ -169,6 +172,9 @@ The |sensor_sampler| samples sensors periodically, according to the configuratio
 Sampling of the sensors is done from a dedicated preemptive thread.
 You can change the thread priority by setting the :kconfig:`CONFIG_CAF_SENSOR_SAMPLER_THREAD_PRIORITY` Kconfig option.
 Use the preemptive thread priority to make sure that the thread does not block other operations in the system.
+The sensor sampler module limits number of ``sensor events`` for each sensor that are submitted, but not processed to prevent OOM error in
+case system workqueue is blocked. For this purpose counter ``active_sensor_events_cnt`` is incremented when sending ``sensor_event``
+and decremented after processing that event. The limit value for maximum unprocessed events for each sensor is placed in ``sensor_config`` in configuration file.
```

Comment:
Decremented while event is processed by the sensor sampler that is final subscriber of the event. 
Because of that the max number of allocated sensor events for given sensor equals active_sensor_events_cnt + 1